docs/components/concepts/clusters.md

```diff
@@ -6,20 +6,20 @@ description: "Learn more about the clusters available in your Camunda Platform 8
 
 A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
 
-Professional and Enterprise plan customers can create as many production or development clusters as they want.
+Starter and Enterprise plan customers can create production or development within the limits of their subscription.
```

Comment:
Is there a link to the marketing website for more subscription info?

Reply:
@akeller we could use the link to the limits that will be something like https://camunda.com/legal/fair-usage-limits-starter-plan. WDYT?

Reply:
Whoops, I missed this. I'll add this link. It goes live on Sept 25, right?

Reply:
```suggestion
Starter and Enterprise plan customers can create production or development [within the limits of their subscription](https://camunda.com/legal/fair-usage-limits-starter-plan).
```